packages/patterns/src/patterns/patternMatchers.js

```diff
@@ -1804,12 +1805,17 @@ const makeMethodGuardMaker = (
     },
   });
 
-const InterfaceGuardShape = harden({
-  klass: 'Interface',
-  interfaceName: M.string(),
-  methodGuards: M.recordOf(M.string(), MethodGuardShape),
-  sloppy: M.boolean(),
-});
+const InterfaceGuardShape = M.splitRecord(
+  {
+    klass: 'Interface',
+    interfaceName: M.string(),
+    methodGuards: M.recordOf(M.string(), MethodGuardShape),
+    sloppy: M.boolean(),
```

Comment:
Just be aware that as of @michaelfig 's upcoming PR, `sloppy` will move into that optional section, and another will appear there as well. No change suggested.